test/integration/service_users_ft_test.js

```diff
@@ -1,15 +1,24 @@
+'use strict'
+
+// NPM modules
 const path = require('path')
 const nock = require('nock')
-const session = require(path.join(__dirname, '/../test_helpers/mock_session.js'))
-const getApp = require(path.join(__dirname, '/../../server.js')).getApp
 const supertest = require('supertest')
-const serviceFixtures = require(path.join(__dirname, '/../fixtures/service_fixtures'))
-const userFixtures = require(path.join(__dirname, '/../fixtures/user_fixtures'))
-const paths = require(path.join(__dirname, '/../../app/paths.js'))
 const csrf = require('csrf')
 const chai = require('chai')
-const expect = chai.expect
 const chaiAsPromised = require('chai-as-promised')
+
+const {expect} = chai
+
+// Local modules
+const session = require(path.join(__dirname, '/../test_helpers/mock_session.js'))
+const {getApp} = require(path.join(__dirname, '/../../server.js'))
+const serviceFixtures = require(path.join(__dirname, '/../fixtures/service_fixtures'))
+const userFixtures = require(path.join(__dirname, '/../fixtures/user_fixtures'))
+const paths = require(path.join(__dirname, '/../../app/paths.js'))
```

Comment:
is our standard to use the `path.join(__dirname...` stylee or not? Seems pointless, and is easier to navigate around without

Reply:
Oh yeah nice spot dont need any of that